packages/loader/container-loader/src/containerContext.ts

```diff
@@ -202,8 +202,8 @@ export class ContainerContext implements IContainerContext {
         return this.container.attachState;
     }
 
-    public createSummary(): ISummaryTree {
-        return this.runtime.createSummary();
+    public createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree {
+        return (this.runtime as any).createSummary(blobRedirectTable) as ISummaryTree;
```

Comment:
It would be great to avoid casting here. I assume container definition needs to be changed first and versions bumped to make it happen, right?